source/features/always-show-all-files.js

```diff
@@ -0,0 +1,10 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+
+export default async function () {
+	const button = select('.js-details-target');
```

Comment:
This selector is too generic. However I'm not seeing the button anymore, did GitHub remove it already?

Reply:
Yep, it is temporal.
https://twitter.com/clarkbw/status/1037729090101334017